src/olympia/signing/tests/test_views.py

```diff
@@ -299,6 +299,14 @@ def test_raises_response_code(self):
             response = self.request('PUT', self.url(self.guid, '1.0'))
             assert response.status_code == 400
 
+    def test_no_version_upload_for_admin_disabled_addon(self):
+        addon = Addon.objects.get(guid=self.guid)
+        addon.update(status=amo.STATUS_DISABLED)
+
+        response = self.request(
+            'PUT', self.url(self.guid, '3.0'), version='3.0')
+        assert response.status_code == 400
+
```

Comment:
Can we have an assertion on the correct error message here too? This will make sure this fails the way we expect it to fail and not because, say, `version` gets removed as a possible argument in the future and this still succeeds but shouldn't.